sos/report/plugins/grafana.py

```diff
@@ -19,29 +19,52 @@ class Grafana(Plugin, IndependentPlugin):
 
     packages = ('grafana',)
 
+    def _is_snap_installed(self):
+        grafana_pkg = self.policy.package_manager.pkg_by_name('grafana')
+        if grafana_pkg:
+            return grafana_pkg['pkg_manager'] == 'snap'
+        return False
+
     def setup(self):
-        if self.get_option("all_logs"):
-            self.add_copy_spec("/var/log/grafana/*.log*")
+        self._is_snap = self._is_snap_installed()
+        if self._is_snap:
+            grafana_cli = "grafana.grafana-cli"
+            log_path = "/var/snap/grafana/common/data/log/"
+            config_path = "/var/snap/grafana/current/conf/grafana.ini"
+
+            self.add_copy_path("snap info grafana")
```

Comment:
I think this is intended to be `add_cmd_output()`?

Reply:
oops, resolved :)